cvat/apps/engine/task.py

```diff
@@ -203,13 +206,53 @@ def _validate_manifest(manifests, root_dir):
         raise Exception('Invalid manifest was uploaded')
     return None
 
+def _validate_url(url):
```

Comment:
@azhavoro , could you please add some comments? Why do you think it is the proper way to verify remote urls? Did you follow an article?

Reply:
Main idea is allow IPs only from public networks, there are many articles about SSRF mitigation, i.e. https://cheatsheetseries.owasp.org/cheatsheets/Server_Side_Request_Forgery_Prevention_Cheat_Sheet.html